components/GoalsCard/GoalsCard.Styled.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+export const StyledList = styled.ul`
+  padding: 0;
+  margin: 0 0.5rem 0 0.5rem;
+  justify-content: space-between;
+  list-style: none;
+  gap: 1rem;
+`;
+
+export const StyledItem = styled.li`
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px,
+    rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px;
+  margin: 10px 0px 10px 0px;
```

Comment:
Consolidate margin values

```suggestion
  margin: 10px 0;
```